src/rules/comment-syntax.ts

```diff
@@ -155,7 +154,7 @@ export default createRule<Options, MessageIds>({
               context.report({ node: comment, messageId: 'lineCommentCapital' });
             }
 
-            if (lastChar === '.') {
+            if (lastChar === '.' && !comment.value.endsWith('etc.') && !comment.value.endsWith(' ...')) {
```

Comment:
Woo that's a nice improvements :pinched_fingers:

Reply:
Yea indeed. Was annoying that this wasn't allowed cause it's pretty usual to have that.